examples/oauth2/README.md

```diff
@@ -210,12 +208,12 @@ We set the following implementation for authentication in `restapi/implementatio
 - Redirecting to the login page
 
 ```go
-func login(r *http.Request) string {
-	// implements the login with a redirection and an access token
-	var accessToken string
-	wG := r.Context().Value(ctxResponseWriter).(http.ResponseWriter)
-	http.Redirect(wG, r, config.AuthCodeURL(state), http.StatusFound)
-	return accessToken
+func login(r *http.Request) middleware.Responder {
```

Comment:
I think the nicer example here would be by adding a response that is for code 302 and returning that instead.
That keeps it all typed. That response needs to define a location header

Reply:
This one was just to follow our previous discussion on handling redirects _with_ http.Redirect(),
and illustrating usage of the ResponderFunc()... and no more publish sample code with context here.

Of course, what you suggest is to avoid this call altogether and process redirection as a regular 302 response with header, etc. Since this significantly deviates from the "oauth2" example, I suggest this could be a complete dedicated example.